presto-blackhole/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.107-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-blackhole</artifactId>
+    <description>Presto - Black Hole Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
Add a comment:
```
<!-- Presto SPI -->
```
See the POM for `presto-mysql` as an example (we forgot to update `presto-example-http`)